Tests/MetaTests/Fixtures/Article.swift

```diff
@@ -9,18 +9,35 @@ enum WrittingStyle {
     case narrative
 }
 
-enum PublicationState {
-    case draft(date: Date, comments: String?)
-    case published(comments: String?, date: Date)
-    case deleted(date: Date, reason: String?, comments: String?)
-}
-
 struct Article {
     let title: String
     let subtitle: String?
     let state: PublicationState
     let author: Author
     let style: WrittingStyle
+    
+    public enum NestedKeys1 {
+        case nested1
+        case nested2
+        private enum NestedKeys2 {
+            case nested3
+            case nested4
+            enum NestedKeys3 {
+                case nested4
+                case nested5
+            }
+        }
+        enum NestedKeys4 {
+            case nested6
+            case nested7
+        }
```

Comment:
I assume it is handled, but it would be good to have some case here named as one of the cases in `NestedKeys1` to test that names do not conflict/shadow other names.

Reply:
Done, in next PR ;)